core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -137,6 +138,19 @@ abstract class SparkFunSuite
     java.nio.file.Paths.get(sparkHome, first +: more: _*)
   }
 
+  // subclasses can override this to exclude certain tests by name
+  // useful when inheriting a test suite but do not want to run all tests in it
+  protected def excluded: Seq[String] = Seq.empty
+
+  override protected def test(testName: String, testTags: Tag*)(testBody: => Any)
+    (implicit pos: Position): Unit = {
+    if (excluded.contains(testName)) {
+      ignore(s"$testName [enable by remove from $excluded]")(testBody)
```

Comment:
this will make the test name very long, how about just `test name (excluded)`

Reply:
OK